content/en/docs/ambient/install/istioctl/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Install with istioctl
+description: Install Istio with support for ambient mode using the istioctl command line tool.
+weight: 10
+keywords: [istioctl,ambient]
+owner: istio/wg-environments-maintainers
+test: no
```

Comment:
```suggestion
test: yes
```

Reply:
It doesn't really do anything, so I don't think testing adds a lot of value?  Do we need the snips for something else?

Reply:
I feel like anything we have a command that has commands on it, we should test it so that we can catch breakages from istio/istio, unless the feature is experimental. Save ourselves from having parts of docs that will be wrong if changes occur.

Reply:
a noble goal